src/keria/testing/testing_helper.py

```diff
@@ -599,6 +777,27 @@ def mockNowIso8601():
     def mockRandomNonce():
         return "A9XfpxIl1LcIkMhUSCCC8fgvkuX8gG9xK3SM-S8a8Y_U"
 
+    @staticmethod
+    def witnessMsg(hab, msg, sn, witHabs):
```

Comment:
test helper to witness a msg given the hab, the msg, the sn, and the list of habs. this functionality is similar to what some keripy tests do.